configuration.k

```diff
@@ -0,0 +1,228 @@
+
+module CONFIGURATION
+
+imports SOLIDITY-SYNTAX
+
+
+syntax START
+
+
+configuration
+    <k> $PGM:START </k>
+    <exit-code exit=""> 1 </exit-code>
+    // ... 
+    
+    <ethereum>
+        
+        // Solidity execution engine
+        // =========================
+        
+        <solidity> 
+            
+            // Mutable during a single transaction
+            // -----------------------------------
+            
+            <execEngine>
+                <output>          .K  </output>
+                <statusCode>      .K  </statusCode>
+                <callStack>     .List  </callStack>
+                <interimStates>   .K  </interimStates>
+                <touchedAccounts> .K  </touchedAccounts>
+                <currentAccount>  #account(#undef_Int,#undef_Id)  </currentAccount>
+		        <result> .K </result>
+		        <mode> .K </mode>
+
+                <callState>
+                
+                    // Should be loaded from <account>
+                    <program> .K </program>              // currently running <contract>
+
+                    // I_*
+                    <id>        0          </id>         // Currently executing contract
+                    <caller>    0          </caller>     // Contract that called current contract
+                    <callData>  .K         </callData>   // .WordStack Copy of register arguments (?)
+                    <callValue>  0          </callValue>  // Value in funds passed to contract
+
+                    // \mu_*
+                    // env: map from qualified name to either storage or local memory
+                    // QName ::= Id | Id.Id /* struct member */ | ... 
+                    // Reference ::= storage(Index, Offset)  // Index, Offset are Word a.k.a. uint256
+                    //             | memory(Index, Offset)
+                    <env> .Map </env>     // Map of names to references
+                
+                    // tenv: map from qualified name to type
+                    // where Type ::= uintXX ... | array | struct(members) | ...
+                    <tenv> .Map </tenv>         // Map names to types
+                
+                    // memory: same as evm!!
+                    // exact allocation strategy may vary(unlike storage), but anyway semantics should be indentical
+                    <localMem> .Map </localMem> 
+                    
+                    // Gas: not sure how, but I think we need it. 
+                    <gas>         .K          </gas>
+                    <memoryUsed>  .K          </memoryUsed>
+                    <previousGas> .K          </previousGas>
+
+                    // Those are present in both KEVM and IELE. 
+                    // Not sure we need them, let's keep them around until figure it out. 
+                    <static>    .K     </static>
```

Comment:
This should start as `false`, not as `.K`. You will need this eventually, it marks whether we're in a call which is not allowed to change the storage state of any accounts.